tests/jobs/test_scheduler_job.py

```diff
@@ -3921,7 +3921,7 @@ def test_do_schedule_max_active_runs_upstream_failed(self):
             schedule_interval='@once',
             max_active_runs=1,
         ) as dag:
-            # Can't use DummyOperator as that goes straight to success
+            # Can't use NoOpOperatoras that goes straight to success
```

Comment:
```suggestion
            # Can't use NoOpOperator as that goes straight to success
```
And in other places - it seams to be badly performed replacement. In general I would suggest to separate this PR into two PRs:
- replace non-inclusive words
- rename `DummyOperator` - this change is a big one and we should probably first reach consensu on the new name

Reply:
Agree with @turbaszek